src/wizard/flutter/index.md

```diff
@@ -88,4 +88,20 @@ Future<void> processOrderBatch(ISentrySpan span) async {
 }
 ```
 
-Check out [the documentation](https://docs.sentry.io/platforms/flutter/performance/instrumentation/) to learn more about the API and automatic instrumentations.
+Check out [the documentation](/platforms/flutter/performance/instrumentation/) to learn more about the API and automatic instrumentations.
+
+## Debug Symbols
+
+Debug symbols provide information that Sentry displays on the **Issue Details** page to help you triage an issue. We offer a range of methods to provide Sentry with debug symbols.
```

Comment:
```suggestion
We offer a range of methods to provide Sentry with debug symbols so that you can see symbolicated stack traces and triage issues faster.
```